syntax = "proto3";

package paxosrpc;

service PaxosRPC {
  rpc Heartbeat (HeartbeatRequest) returns (HeartbeatReply) {}
}

message HeartbeatRequest {
  int32 min_last_executed = 1;
}

message HeartbeatReply {
  int32 last_executed = 1;
}
